### DISCUSSION TITLE
The control diagrams and differential equations of models

### DISCUSSION THREAD

**Discussion by Tang115** [CURENT/andes#231] [General]
Dr. Cui

I have the following questions:

1) Is there a document/source which summarizes the control diagrams of models (e.g. PSS, Exciter, and etc.) in Andes?

2) I am wondering that why TGOV1's differential equations look different in 3.10.1 and 8.31.2 of the ANDES Manual.

Thanks!
Best regard

**Comment by cuihantao** [comment]
Most models in ANDES are the same as that from PSS/E or Powerworld. You can search the model name and "Powerworld" to obtain the diagram. 

TGOV1 was recently revised to use the convention in Powerworld and PSS/E. The figure in 3.10.1 is outdated. In that version, \Delta \omega was defined as (1 - omega), and now \Delta \omega is \omega - 1. Both models are correct.
